### ISSUE TITLE
Report Submission Confirmation

### ISSUE THREAD

**Issue by jakepeyser** [yangweichen/inspection_app#10]

**Comment by rhgrant7** [comment]
If being used with auth/pulling phone number from phone, no need for signature. Will confirm with customers, but most likely.

**Comment by jakepeyser** [comment]
Yep, we can view sigs as a future addition if asked for by fleet operators
